### PR TITLE
updating caption

### LINKED ISSUE #5: Improve the README
## Do you have a favorite Git Resource?
Add it to the README!
 If you need help, check out the Appendix in the training manual.

### PR DESCRIPTION
**Pull request by Enoch-Israel**
This closes #5 

## Changes

- This is a new change
- That fixes slide 04

### CODE REVIEW

File: _slides/04-Enoch-Israel.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Caption here Modified
```
Comment (Enoch-Israel) [inline _slides/04-Enoch-Israel.md:6]: Not a very good modification to the file.

**Review by Enoch-Israel — COMMENTED** [review]
Added comment

File: _slides/04-Enoch-Israel.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
```
Comment (ootbutterfus) [inline _slides/04-Enoch-Israel.md:6]: looks good

**Review by ootbutterfus — APPROVED** [review]
completed!

**Pull request merged**